lib/hyrax/rails/routes.rb

```diff
@@ -46,18 +46,9 @@ def curation_concerns_basic_routes(&block)
       end
     end
 
-    # kmr added :show to make tests pass
+    Deprecation.warn(self, "#curation_concerns_embargo_management is deprecated and has no effect. It will be removed in Hyrax 3.0")
```

Comment:
This has to be inside the body of the `curation_concerns_embargo_management` method.